src/shared/components/query/quickSearch/QuickSearch.tsx

```diff
@@ -55,6 +61,16 @@ export default class QuickSearch extends React.Component {
     @observable private inputValue: string = '';
     @observable private isFocusing = false;
     @observable private isLoading = false;
+    private studies: Map<string, string>;
+
+    constructor(props: QuickSearchProps) {
+        super(props);
+
+        this.studies = props.studies.reduce((prev, curr) => {
```

Comment:
This is a bit of an anti-pattern to do this in contructor.  If for some reason the property were to change, we would want this list to recompute.  By doing this in constructor, it would become stale.  In this particular case, it will never happen b/c studies are essentially static, but good to get it right.   So, this should just be a computed property